pkg/ingester/ingester.go

```diff
@@ -127,6 +129,11 @@ func (i *Ingester) Shutdown() {
 	i.lifecycler.Shutdown()
 }
 
+// Stopping helps cleaning up resources before actual shutdown
+func (i *Ingester) Stopping() {
+	close(i.quitting)
```

Comment:
why not call close on all tailer here and drop the quitting chan ? seems that tailer has already a done chan that you can use using `t.close` ?

Reply:
Actually ingester does not have references to open tailers, thats why I did it like that. Should I add the code to store the references and call close on them?

Reply:
but you could do close on stream that close on tailer and close everything string ?

Reply:
Calling close tailers on all the instances, they hold a reference to all the tailers associated with them.